_plauscherl/63.html

```diff
@@ -25,4 +25,8 @@
   name: Karina Streng
   talk: "Leadership"
   img:  "/img/speaker_female.jpg"
+-
+  name: Paul Lanzerstorfer
+  talk: Agile Softwareentwicklung in der Linzer IT-Community
+  img: //www.dropbox.com/s/b01m66swkdvitce/me.jpg
```

Comment:
This link points to a Website that contians the image - you can add a tinified (tinyjpg.com) version of your image to the commit (see /img/speaker-avatars/* )